test/integration/features/config.feature

```diff
@@ -9,67 +9,99 @@ Checks whether CRC `config set` command works as expected in conjunction with `c
 
         @linux
         Examples: Config settings on Linux
-            | property  |  value1 |   value2 |
-            | cpus      |       4 |        3 |
-            | memory    |    8192 |     4096 |
-            | vm-driver | libvirt | hyperkit |
+            | property         |    value1 |            value2 |
+            | cpus             |         4 |                 3 |
+            | memory           |      8192 |              4096 |
+            | vm-driver        |   libvirt |          hyperkit |
+            | nameserver       | 120.0.0.1 |   999.999.999.999 |
+            | pull-secret-file |      /etc | /nonexistent-file |
 
         @windows
         Examples: Config settings on Windows
-            | property  | value1                   | value2                                               |
-            | cpus      | 4                        | 3                                                    |
-            | memory    | 8192                     | 4096                                                 |
-            | vm-driver | hyperv                   | hyperkit                                             |
+            | property         |    value1 |            value2 |
+            | cpus             |         4 |                 3 |
+            | memory           |      8192 |              4096 |
+            | vm-driver        |    hyperv |          hyperkit |
+            | nameserver       | 120.0.0.1 |   999.999.999.999 |
+            | pull-secret-file |    /Users | /nonexistent-file |
 
     @linux @darwin @windows
     Scenario: CRC config checks (bundle version)
         When setting config property "bundle" to value "current bundle" succeeds
         And "JSON" config file "crc.json" in CRC home folder contains key "bundle" with value matching "current bundle"
         And setting config property "bundle" to value "/path/to/nonexistent/bundle/crc_hypervisor_version.tar.xz" fails
 
+    @linux @darwin @windows
+    Scenario: CRC config checks (update check)
+        Given setting config property "disable-update-check" to value "true" succeeds
+        When executing "crc setup" succeeds
+        Then stdout should not contain "new version" 
+        And "JSON" config file "crc.json" in CRC home folder contains key "disable-update-check" with value matching "true"
+        And unsetting config property "disable-update-check" succeeds
+
 # WARNINGS
 
     Scenario Outline: CRC config checks (warnings)
         When setting config property "<property>" to value "<value1>" succeeds
         Then "JSON" config file "crc.json" in CRC home folder contains key "<property>" with value matching "<value1>"
         When setting config property "<property>" to value "<value2>" succeeds
         Then "JSON" config file "crc.json" in CRC home folder contains key "<property>" with value matching "<value2>"
+        And unsetting config property "<property>" succeeds
```

Comment:
`And "JSON" config file "crc.json" in CRC home folder does not contain key "<property>"`
Can this statement work at the moment? Or would it need some changes in crcsuite.go?

Reply:
Thanks. I assumed I hadn't implemented the `does not contain` option in https://github.com/code-ready/clicumber/blob/02a92c8f7eceabab30fac8c66ef7d93c6a2c3f5c/testsuite/testsuite.go#L139, but it's there. Thanks. I now updated the feature file to check the crc.json file after `crc config unset`.